pkgs/applications/radio/ebook2cw/Makefile.patch

```diff
@@ -1,8 +1,29 @@
 diff --git a/Makefile b/Makefile
-index 005d60d..f69c7fe 100644
+index 005d60d..0718f7e 100644
 --- a/Makefile
 +++ b/Makefile
-@@ -41,6 +41,7 @@ install:
+@@ -24,23 +24,24 @@ endif
+ all: ebook2cw
+ 
+ ebook2cw: ebook2cw.c codetables.h
+-	gcc ebook2cw.c -pedantic -Wall -lm $(LDFLAGS) $(CFLAGS) -o ebook2cw
++	$(CC) ebook2cw.c -pedantic -Wall -lm $(LDFLAGS) $(CFLAGS) -o ebook2cw
```

Comment:
do you mind upstreaming this if possible? doesn't seem to be a nix-specific patch

Reply:
The issue tracker appears to be private. I guess I'll mail them to make them aware of our patches. 😕

Reply:
Good thing is: upstream is super responsive and merged it within 15 minutes. :)